traffic_ops/v5-client/server_server_capabilities.go

```diff
@@ -31,14 +31,14 @@ const apiServerServerCapabilities = "/server_server_capabilities"
 // apiMultipleServersCapabilities is the API version-relative path to the /multiple_servers_capabilities API endpoint.
 const apiMultipleServersCapabilities = "/multiple_servers_capabilities"
 
-// CreateServerServerCapability assigns a Server Capability to a Server.
-func (to *Session) CreateServerServerCapability(ssc tc.ServerServerCapability, opts RequestOptions) (tc.Alerts, toclientlib.ReqInf, error) {
+// CreateServerServerCapabilityV5 assigns a Server Capability to a Server.
+func (to *Session) CreateServerServerCapability(ssc tc.ServerServerCapabilityV5, opts RequestOptions) (tc.Alerts, toclientlib.ReqInf, error) {
```

Comment:
`CreateServerServerCapabilityV5` description doesn't match `CreateServerServerCapability` func name

Reply:
Changed godoc to `CreateServerServerCapability` in c2e7652089